qc-runner/src/dataIntegration.r

```diff
@@ -115,6 +115,13 @@ run_dataIntegration <- function(scdata, config){
         scdata <-Seurat::FindVariableFeatures(scdata, selection.method = "vst", nfeatures = nfeatures, verbose = F)
     }
 
+    scdata <- FindVariableFeatures(scdata, selection.method = "vst", assay = "RNA", nfeatures = nfeatures, verbose = FALSE)
```

Comment:
This is calculated above, wouldn't that work?

Reply:
Good point, I guess this line belongs after line 111, also line 115 should then also have `assay = "RNA"`?

Reply:
Yes, in order to have the dispersion, we need to compute with RNA, since with integrated it fails

Reply:
@JuanluOnieva-biomage , basically we can then delete line 118, since line 121 is run `FindVariableFeatures` regardless of uni/multi-sample?